pkg/kn/commands/source/apiserver/apiserver_test.go

```diff
@@ -80,11 +80,15 @@ func cleanupAPIServerMockClient() {
 	apiServerSourceClientFactory = nil
 }
 
-func createAPIServerSource(name, resourceKind, resourceVersion, serviceAccount, mode string, ceOverrides map[string]string, sink duckv1.Destination) *v1.ApiServerSource {
-	resources := []v1.APIVersionKindSelector{{
-		APIVersion: resourceVersion,
-		Kind:       resourceKind,
-	}}
+func createAPIServerSource(name, serviceAccount, mode string, resourceKind, resourceVersion []string, ceOverrides map[string]string, sink duckv1.Destination) *v1.ApiServerSource {
```

Comment:
Whenever I see a function with so many parameters (in code or test) I wonder whether a struct should be created to hold the related params and the struct passed to the func or be a method?